### PR TITLE
use new .csproj for multitarget with .net35 and netstandard

### PR DESCRIPTION
**Pull request by paulpach**

### CODE REVIEW

**Comment by vis2k** [comment]
works in Rider so far.
but looks like the debug-editor and release-editor build targets are still missing

File: Telepathy.Tests/Server2ClientTest.cs [hunk 1]
```diff
@@ -31,7 +28,7 @@ public void TestServer()
             Message connectMsg = NextMessage(client);
             Assert.That(connectMsg.eventType, Is.EqualTo(EventType.Connected));
 
-            uint id = connectMsg.connectionId;
+            int id = connectMsg.connectionId;
```
[the remark this answers is not in the document]
Comment (paulpach) [reply Telepathy.Tests/Server2ClientTest.cs]: yes and no.

In master,  this file is there but it is being ignored completely.    I have not looked into if this test is redundant or not yet.

**Comment by paulpach** [comment]
we only did the debug-editor release-editor thing for Mirror,  we don't need it for Telepathy

File: Telepathy.Tests/Server2ClientTest.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (paulpach) [reply Telepathy.Tests/Server2ClientTest.cs]: This test is redundant,  It is not being run in master at all,  and it duplicates (poorly)  a test already present in TransportTest.cs.   I removed this file in master and merged here.

**Pull request merged**